tasks/22_remove-falsy-values/remove-falsy-values.js

```diff
@@ -0,0 +1,6 @@
+export function removeFalsyValues(arr) {
+  return arr.filter((el) => {
+    if (el) return true;
+    return false;
```

Comment:
I think that this can be simplified a lot and downsized to one line and that it will become even more readable.

Reply:
I think I figured it out and implemented the change.